cmd/jujud-controller/agent/agenttest/agent.go

```diff
@@ -236,6 +237,7 @@ func (s *AgentSuite) PrimeStateAgentVersion(c *gc.C, tag names.Tag, password str
 	err = database.BootstrapDqlite(
 		context.Background(),
 		database.NewNodeManager(conf, true, logger, coredatabase.NoopSlowQueryLogger{}),
+		model.MustNewUUID(),
```

Comment:
Not a fan of the possible panic.

Reply:
These are in tests :shrug:

Reply:
How do you mean? This isn't a test right ?